lib/hybridDB.ts

```
@@ -296,6 +309,52 @@ export class HybridDB {
+      console.error('Failed to load priority threads from remote:', error);
+      return [];
+    }
+  }
+
+  // Load regular threads with pagination from remote
+  static async loadRegularThreadsPaginated(limit: number = 25, offset: number = 0): Promise<{
+    threads: Thread[];
+    hasMore: boolean;
+    total: number;
+  }> {
+    if (this.isGuestMode) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)